### PR TITLE
Create 2024-03-22_mooc_cubeandclouds.md

### PR DESCRIPTION
**Pull request by przell**
News item on the MOOC Cubes and Clouds

### CODE REVIEW

**Comment by m-mohr** [comment]
Could you provide a bit more context and how this relates to openEO?

**Comment by przell** [comment]
@m-mohr, good point! I've added the info here: https://github.com/Open-EO/openeo.org/pull/86/commits/a69f49ad0ae4646b821686b11b51dbdd30c87b45

File: news/2024-03-22_mooc_cubeandclouds.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+news:   true
+title:  MOOC Cubes and Clouds open for registration
+date:   2024-03-22
+author: Peter Zellner
+---
+
+**MOOC Cubes and Clouds**
```
Comment (m-mohr) [inline news/2024-03-22_mooc_cubeandclouds.md:8]:
```suggestion
```

File: news/2024-03-22_mooc_cubeandclouds.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+news:   true
+title:  MOOC Cubes and Clouds open for registration
```
Comment (m-mohr) [inline news/2024-03-22_mooc_cubeandclouds.md:3]:
```suggestion
title:  MOOC Cubes and Clouds - open for registration
```

File: news/2024-03-22_mooc_cubeandclouds.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+date:   2024-03-22
+author: Peter Zellner
+---
+
+**MOOC Cubes and Clouds**
+
+:cloud: Dive into the world of data cubes, cloud platforms, and open science with our latest EO college course “Cubes and Clouds”
```
Comment (m-mohr) [inline news/2024-03-22_mooc_cubeandclouds.md:10]:
```suggestion
The latest EO college course “Cubes and Clouds” explores the newest concepts in Earth Observation through engaging lectures, videos, animated content, and hands-on exercises!
```

File: news/2024-03-22_mooc_cubeandclouds.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+title:  MOOC Cubes and Clouds open for registration
+date:   2024-03-22
+author: Peter Zellner
+---
+
+**MOOC Cubes and Clouds**
+
+:cloud: Dive into the world of data cubes, cloud platforms, and open science with our latest EO college course “Cubes and Clouds” 
+
+:rocket: Explore the newest concepts in Earth Observation through engaging lectures, videos, animated content, and hands-on exercises!
```
Comment (m-mohr) [inline news/2024-03-22_mooc_cubeandclouds.md:12]:
```suggestion
```

File: news/2024-03-22_mooc_cubeandclouds.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+:computer: openEO plays a main role in the course. It's standardized API is homogenizing cloud processing. It allows for portablility of the created workflow between different cloud providers, reproducibility and sharing of the workflow which is enriched with valuable metadata. A great step towards open science.
```
Comment (m-mohr) [inline news/2024-03-22_mooc_cubeandclouds.md:13]:
```suggestion
openEO plays a main role in the course due to its standardized API for homogenizing cloud processing. It allows for portablility of the created workflow between different cloud providers, reproducibility and sharing of the workflow which is enriched with valuable metadata. The idea behind and need for openEO are tought in a dedicated lecture. All hands-on exercises use openEO, guiding participants from basic processes to carrying out end-to-end EO workflows and sharing their results!
```

File: news/2024-03-22_mooc_cubeandclouds.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+**MOOC Cubes and Clouds**
+
+:cloud: Dive into the world of data cubes, cloud platforms, and open science with our latest EO college course “Cubes and Clouds” 
+
+:rocket: Explore the newest concepts in Earth Observation through engaging lectures, videos, animated content, and hands-on exercises!
+
+:computer: openEO plays a main role in the course. It's standardized API is homogenizing cloud processing. It allows for portablility of the created workflow between different cloud providers, reproducibility and sharing of the workflow which is enriched with valuable metadata. A great step towards open science. 
+
+:books: The idea behind and need for openEO are tought in a dedicated lecture. All hands-on exercises use openEO, guiding participants from basic processes to carrying out end-to-end EO workflows and sharing their results!
```
Comment (m-mohr) [inline news/2024-03-22_mooc_cubeandclouds.md:15]:
```suggestion
```

File: news/2024-03-22_mooc_cubeandclouds.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+
+:exclamation: Subscribe and become a cloud native EO expert :point_right: [Cubes & Clouds – EO College (eo-college.org)](https://eo-college.org/courses/cubes-and-clouds/)
```
Comment (m-mohr) [inline news/2024-03-22_mooc_cubeandclouds.md:17]:
```suggestion
You can subscribe to the course for free and learn about cloud native EO at [Cubes & Clouds – EO College (eo-college.org)](https://eo-college.org/courses/cubes-and-clouds/).
```

**Comment by m-mohr** [comment]
Thanks for the updates. I tried to make it a little less marketing-style. This read like a placed advertisment.

**Comment by przell** [comment]
Great, thanks for adding the news item!!!!!!

**Pull request merged**